src/views/login/tabs/SignUpTab.tsx

```diff
@@ -4,48 +4,91 @@ import { FaUserCheck } from "react-icons/fa";
       <hgroup className="flex flex-col items-center w-full">
         <h2 className="text-4xl font-bold">Register</h2>
         <p className="mt-2">Setup a new account in a minute.</p>
       </hgroup>
 
-      <div className="mt-11">
-        <FormInputWithTooltip
-          placeholder="Phone number"
-          tooltip="Please follow this example - 01XXXXXXXXX"
-        />
-        <FormInputWithTooltip
-          placeholder="Password"
-          tooltip="Password must be 6 characters"
-        />
-        <FormInputWithTooltip
-          placeholder="Repeat Password"
-          tooltip="Password must be 6 characters"
-        />
-      </div>
-
-      <div className="mt-5 flex">
-        <FormCheckboxInput
-          label={
-            <>
-              I agree to the all{" "}
-              <Link className="text-[#007bff]" to="/">
-                terms & consitions
-              </Link>{" "}
-              of bebostha.
-            </>
-          }
-        />
-      </div>
-
-      <FormSubmit>
-        <FaUserCheck />
-        <p className="ml-2">Create new account</p>
-      </FormSubmit>
+      <form onSubmit={handleSubmit(onSubmit)}>
+        <div className="mt-11">
+          <FormInputWithTooltip
+            id="phone"
+            placeholder="Phone number"
+            register={register}
+            options={{
+              required: "This is required",
+              minLength: {
+                value: 4,
+                message: "Min length is 4",
+              },
+            }}
+            tooltip={errors.phone && errors.phone.message}
+          />
+          <FormInputWithTooltip
+            id="password"
+            placeholder="Password"
+            register={register}
+            options={{
+              required: "This is required",
+              minLength: {
+                value: 8,
+                message: "Min length is 8",
+              },
+            }}
+            tooltip={errors.password && errors.password.message}
+          />
+          <FormInputWithTooltip
+            id="repeatPassword"
+            placeholder="Repeat Password"
+            register={register}
+            options={{
+              required: "This is required",
+              minLength: {
+                value: 8,
+                message: "Min length is 8",
+              },
+            }}
+            tooltip={errors.repeatPassword && errors.repeatPassword.message}
+          />
+        </div>
+
+        <div className="mt-5 flex">
+          <FormCheckboxInput
+            label={
+              <>
+                I agree to the all{" "}
+                <Link className="text-[#007bff]" to="/">
+                  terms & consitions
+                </Link>{" "}
+                of bebostha.
+              </>
+            }
+          />
+        </div>
+
+        <FormSubmit>
+          <FaUserCheck />
+          <p className="ml-2">Create new account</p>
+        </FormSubmit>
+      </form>
```

Comment:
I would like you to do inputs for SignUp.
Name
Email
Password
Confirm Password